index.html

```diff
@@ -1238,9 +1238,28 @@ <h4>TTML Format</h4>
       <p class="issue" data-number="25"></p>
       <section>
         <h4>Proprietary Metadata</h4>
-        <p>Many dubbing and <a>audio description</a> workflows permit annotation of <a>Script Events</a> or documents with proprietary metadata.
-          Additional vocabulary MAY be included if it is in namespaces not defined by this specification, [[TTML2]] or [[ttml-imsc1.2]].</p>
-        <p class="issue" data-number="31"></p> 
+        <p>Many dubbing and <a>audio description</a> workflows permit annotation of <a>Script Events</a> or documents with proprietary metadata. Exsiting metadata vocabulary from [[TTML2]] MAY be included. Additional vocabulary in proprietary namespaces MAY also be included.</p>
```

Comment:
typo, plus it'd be really helpful for future editing reviews if we could keep lines shorter and break them at grammatically useful places:
```suggestion
        <p>Many dubbing and <a>audio description</a> workflows permit annotation of <a>Script Events</a>
        or documents with proprietary metadata.
        Existing metadata vocabulary from [[TTML2]] MAY be included.
        Additional vocabulary in proprietary namespaces MAY also be included.</p>
```

(the typo fix is `exsiting` -> `existing`)

Also:
* need to be clear what a "proprietary" namespace is, and 
* mention `daptm` namespace metadata defined in this spec.

Reply:
This typo came back in somehow.

Reply:
fixed

Reply:
At some point we should do a big trawl through for line breaks too.